_specifications/specification-3-17.md

```diff
@@ -6241,6 +6262,13 @@ export namespace CodeActionKind {
  * a code action is run.
  */
 export interface CodeActionContext {
+	/**
+	 * The reason why code actions were requested.
```

Comment:
should we define what the absence of this value means. e.g. that the server should make no assumptions about the kind of trigger and return all code actions that are relevant ?

Reply:
Makes sense. But it is also somehow implicit since we didn't have that value before and servers needed to return all values.